src/vm/methodtablebuilder.cpp

```diff
@@ -5120,7 +5120,7 @@ MethodTableBuilder::InitNewMethodDesc(
     }
 
     // Check for methods marked as [Intrinsic]
-    if (GetModule()->IsSystem())
+    if (GetModule()->IsSystem() || GetAssembly()->IsSIMDVectorAssembly())
```

Comment:
This is the same check we do for the types: https://github.com/dotnet/coreclr/blob/7568519bf2a964139d39932534569bf5299347cb/src/vm/methodtablebuilder.cpp#L1527